src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -29,10 +31,14 @@ export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
   return (
     <div className="admin-venue">
       <div className="admin-venue__header">
-        <div className="admin-venue__title">Admin Dashboard</div>
-        <Button as={Link} to="/admin-ng/create/venue">
-          Create a new space
-        </Button>
+        <div className="admin-venue__title">
+          <M>{AdminVenues.name}.title</M>
+        </div>
+        <M>
```

Comment:
Ignores complex children. Works only with strings and arrays (should probably be a check it's an array of all strings though